spec/unit/knife/bootstrap_options_spec.rb

```diff
@@ -148,12 +153,17 @@ def compare_property(sym)
       :ssh_gateway_identity,
       :bootstrap_proxy_user,
       :bootstrap_proxy_pass,
+      :bootstrap_preinstall_command,
+      :distro, # Deprecated - remove this when the flag is removed.
+      :template_file, # Deprecated - remove this when the flag is removed.
     ]}
     # win_ignore: Options in windows that aren't relevant to core.
     let(:win_ignore) { [
       :auth_timeout,
       :install_as_service,
       :host_key_verification,  # Deprecated - remove this when the flag is removed.
+      :distro, # Deprecated - remove this when the flag is removed.
+      :template_file, # Deprecated - remove this when the flag is removed.
```

Comment:
i think these two aren't supposed to be here? options shouldn't be in both the `ref_ignore` and `win_ignore`.

Reply:
Hi @btm, as per my first commit `distro` & `template_file` options was only added in `win_ignore` & locally it was working fine for all 10 example successfully on `chef 14` & `chef 13` as well. But travis was getting fail on `chef 13` for example https://github.com/MsysTechnologiesllc/knife-windows/blob/d3130b14110e2276c6bcd1b8db8faf2b5a71081e/spec/unit/knife/bootstrap_options_spec.rb#L47, I was not able reprodcue travis issue locally so I have added it in `ref_ignore` too to pass the travis.